engine/src/physics/solver/integration/physics_constants_integration.hpp

```diff
@@ -0,0 +1,24 @@
+/// @file
+/// @brief Resource @ref cubos::engine::MagiConfig.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Resource which allows configuration over magic numbers in the solver plugin.
+    /// @ingroup physics-plugin
+    struct PhysicsConstantsIntegration
+    {
+        CUBOS_REFLECT;
+
+        float cmpInvMass = 0.0F;
+
+        float cmpInvInertia = 0.0F;
+    };
```

Comment:
Since these are both the minimum values, instead of `cmp` we should have `min` in the name, it's a bit clearer.